full-service/src/db/txo.rs

```diff
@@ -639,23 +635,52 @@ impl TxoModel for Txo {
     }
 
     fn list_unspent(
-        account_id_hex: &str,
+        account_id_hex: Option<&str>,
         assigned_subaddress_b58: Option<&str>,
         token_id: Option<u64>,
         offset: Option<u64>,
         limit: Option<u64>,
         conn: &Conn,
     ) -> Result<Vec<Txo>, WalletDbError> {
-        use crate::db::schema::txos;
+        use crate::db::schema::{transaction_input_txos, transaction_logs, txos};
+
+        /*
+            SELECT * FROM txos
```

Comment:
Do we need to save this raw SQL comment?

Reply:
I think it's nice to have the raw SQL query for reference, as reading the diesel stuff can be a little confusing if you're not familiar with it. Especially with filter chaining, the rules for that are very particular and diesel does some weird stuff sometimes that's unexpected.